### ISSUE TITLE
Focus Target, RaidFrames, Castbar, Bags Broken.

### ISSUE THREAD

**Issue by KalmTraveler** [Kkthnx/KkthnxUI#16]
Hi im a great fan of your interface so i mainly use it on my prot warrior to tank. But there are a few things broken since the last patch.

Before the patch i was able to set an focus target by shit+left clicking my target. This seems to be gone -.-
Also there is now the classic raid and group frame there there was a custom before (same with bags)
On the Castbar (which moved the place again -.-) is now an icon which i cannot remove and also a little timer.
The item level display in the character sheet does not update with item upgrades so 730 items display still as 720 items.

edit: Is there a way opening the calender?

Long story short: Please update and give us more options. Moving Cast Bar would be an decent setting also the buff and cooldown thing the only way i found out to disable it was by removing the Filger addon...

**Comment by Kkthnx** [comment]
I havent been on retail in some time but I will get on this as soon as possible.

**Comment by jenderle7** [comment]
Any possible update on this? Love this UI!

**Comment by KalmTraveler** [comment]
so this is dead?

**Comment by Kkthnx** [comment]
Not dead. This is on my list of things to look into.
